### PR TITLE
[Hub Generated] Review request for Microsoft.AppPlatform to add version stable/2020-07-01

### PR DESCRIPTION
**Pull request by bowen5**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/bowen5/azure-rest-api-specs/dev-appplatform-Microsoft.AppPlatform-2020-07-01...master/).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for AME onboarding.
- [x] [Validation tools](https://aka.ms/swaggertools) have been run on swagger spec(s) updated and errors have all been fixed with this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, [please view the FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist

Private PR has been signed off by ARM: https://github.com/Azure/azure-rest-api-specs-pr/pull/1627

- [x] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories.
*	adding/removing APIs.
* adding/removing properties.
* adding/removing API-version.
* adding a new service in Azure.

*Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.*

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them.

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by bowen5** [comment]
@majastrz , would you please help to sign off this PR? The content of this PR actually has been already signed off in private repo here https://github.com/Azure/azure-rest-api-specs-pr/pull/1627. Thank you!

**Review by majastrz — APPROVED** [review]
Thanks for making the first commit the original API version. It makes it much easier to review. The delta looks good. Signed off from ARM side.

**Review by njuCZ — APPROVED** [review]

**Review by ArcturusZhang — CHANGES_REQUESTED** [review]
Could you please also add a new tag entry in the readme.go.md since you are requesting this package to be released in go SDK?

**Comment by yuwzho** [comment]
> Could you please also add a new tag entry in the readme.go.md since you are requesting this package to be released in go SDK?

@ArcturusZhang Thanks, just updated.

File: specification/appplatform/resource-manager/readme.go.md [hunk 1]
```diff
@@ -13,9 +13,19 @@ go:
...
   - tag: package-2019-05-01-preview
 ```
 
+### Tag: package-2020-07 and go
+
+These settings apply only when `--tag=package-2020-07 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2020-07' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2020-07-01/$(namespace)
```
Comment (ArcturusZhang) [inline specification/appplatform/resource-manager/readme.go.md:26]:
```suggestion
output-folder: $(go-sdk-folder)/services/$(namespace)/mgmt/2020-07-01/$(namespace)
```

**Review by ArcturusZhang — CHANGES_REQUESTED** [review]
Your new api-version should be stable, but the output-folder is accidentally in preview sub-directory

**Review by ArcturusZhang — APPROVED** [review]

**Pull request merged**